mysqld_exporter.go

```diff
@@ -52,6 +63,56 @@ func NewMySQLExporter(dsn string) *Exporter {
 			Help:      "The last scrape error status.",
 		}),
 		metrics: map[string]prometheus.Gauge{},
+		commands: prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: namespace,
+			Name:      "commands_total",
+			Help:      "Number of executed mysql commands.",
+		}, []string{"command"}),
+		connection_errors: prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: namespace,
+			Name:      "connection_errors_total",
+			Help:      "Number of mysql connection errors.",
+		}, []string{"error"}),
+		handlers: prometheus.NewCounterVec(prometheus.CounterOpts{
+			Namespace: namespace,
+			Name:      "handlers_total",
+			Help:      "Number of executed mysql handlers.",
+		}, []string{"handler"}),
+		innodb_buffer_pool: prometheus.NewUntypedVec(prometheus.UntypedOpts{
+			Namespace: namespace,
+			Name:      "innodb_buffer_pool_total",
```

Comment:
Here is a new output from the exporter:
```
# HELP mysql_innodb_buffer_pool_bytes_total The total number of bytes in the InnoDB buffer pool.
# TYPE mysql_innodb_buffer_pool_bytes_total gauge
mysql_innodb_buffer_pool_bytes_total{type="data"} 2.424832e+06
mysql_innodb_buffer_pool_bytes_total{type="dirty"} 0
# HELP mysql_innodb_buffer_pool_pages_total The number of pages in the InnoDB buffer pool.
# TYPE mysql_innodb_buffer_pool_pages_total gauge
mysql_innodb_buffer_pool_pages_total{type="data"} 148
mysql_innodb_buffer_pool_pages_total{type="dirty"} 0
mysql_innodb_buffer_pool_pages_total{type="flushed"} 1
mysql_innodb_buffer_pool_pages_total{type="free"} 8044
mysql_innodb_buffer_pool_pages_total{type="misc"} 0
# HELP mysql_innodb_buffer_pool_read_ahead
# TYPE mysql_innodb_buffer_pool_read_ahead untyped
mysql_innodb_buffer_pool_read_ahead 0
# HELP mysql_innodb_buffer_pool_read_ahead_evicted
# TYPE mysql_innodb_buffer_pool_read_ahead_evicted untyped
mysql_innodb_buffer_pool_read_ahead_evicted 0
# HELP mysql_innodb_buffer_pool_read_ahead_rnd
# TYPE mysql_innodb_buffer_pool_read_ahead_rnd untyped
mysql_innodb_buffer_pool_read_ahead_rnd 0
# HELP mysql_innodb_buffer_pool_read_requests
# TYPE mysql_innodb_buffer_pool_read_requests untyped
mysql_innodb_buffer_pool_read_requests 511
# HELP mysql_innodb_buffer_pool_reads
# TYPE mysql_innodb_buffer_pool_reads untyped
mysql_innodb_buffer_pool_reads 149
# HELP mysql_innodb_buffer_pool_wait_free
# TYPE mysql_innodb_buffer_pool_wait_free untyped
mysql_innodb_buffer_pool_wait_free 0
# HELP mysql_innodb_buffer_pool_write_requests
# TYPE mysql_innodb_buffer_pool_write_requests untyped
mysql_innodb_buffer_pool_write_requests 1
```
I added groups for bytes and pages.
Do you think that more groups are needed?